GVRf/Framework/backend_daydream/src/main/java/org/gearvrf/DaydreamViewManager.java

```diff
@@ -28,52 +28,49 @@
 
 import javax.microedition.khronos.egl.EGLConfig;
 
-class OvrGoogleVRViewManager extends OvrViewManager {
-    private static final String TAG = OvrGoogleVRViewManager.class.getSimpleName();
+class DaydreamViewManager extends GVRViewManager {
+    private static final String TAG = DaydreamViewManager.class.getSimpleName();
     private final float[] headTransform;
 
-    OvrGoogleVRViewManager(final GVRActivity gvrActivity, GVRScript gvrScript, OvrXMLParser
-            xmlParser) {
-        super(gvrActivity, gvrScript, xmlParser);
+    DaydreamViewManager(final GVRActivity gvrActivity, GVRScript gvrScript) {
+        super(gvrActivity, gvrScript);
         GvrView gvrView = new GoogleVRView(gvrActivity, this, null);
         gvrActivity.setContentView(gvrView);
         headTransform = new float[4];
-        mLensInfo.setFBOWidth(VrAppSettings.DEFAULT_FBO_RESOLUTION);
-        mLensInfo.setFBOHeight(VrAppSettings.DEFAULT_FBO_RESOLUTION);
     }
 
     public void onNewFrame(HeadTransform headTransform) {
         headTransform.getQuaternion(this.headTransform, 0);
         mMainScene.getMainCameraRig().getHeadTransform().setRotation(this.headTransform[3],
                 this.headTransform[0], this.headTransform[1], this.headTransform[2]);
+
+        updateSensoredScene();
     }
 
     public void onDrawEye(Eye eye) {
         if (eye.getType() == Eye.Type.LEFT) {
-            OvrMonoscopicRenderer.cull(mMainScene, mMainScene.getMainCameraRig().getCenterCamera
-                    (), mRenderBundle);
-            renderCamera(mActivity.getNative(), mMainScene, mMainScene
-                    .getMainCameraRig().getLeftCamera(), mRenderBundle);
+//            OvrMonoscopicRenderer.cull(mMainScene, mMainScene.getMainCameraRig().getCenterCamera
```

Comment:
Removed commented out code from the daydream viewManager. Culling (and shadow mapping) is done by the base viewManager for all backends.